entity/node/accessors/node_modules/iotAuthService.js

```diff
@@ -458,6 +465,7 @@ eventHandlers = {
 }
 */
 exports.sendSessionKeyReqViaTCP = function(options, sessionKeyRespHandler, eventHandlers, callbackParams) {
+    console.log(options);
```

Comment:
I don't think this is necessary.

Reply:
Sorry for confusing you. The PR wasn't ready yet.

The point I sent a notification for review was before I added the changes in node. I was planning to make a different PR on this, but forgot to make a different branch and kept working on this branch...

I didn't know how to "undo" this notification, but still I should've notify you.

I will review the whole PR again.